docs/conf.py

```diff
@@ -1,8 +1,11 @@
 # -*- coding: utf-8 -*-
-import sys, os, pkginfo, datetime
+import datetime
```

Comment:
Wasted lines? What on Earth are you talking about Chris?!

(Also: If you don't want to see imports at the top of the file, configure your editor to collapse them by default? :-)